src/application/usecases/__tests__/SuccessExampleLibraryUseCase.test.ts

```
@@ -0,0 +1,440 @@
+  vi,
+  beforeEach,
+  type MockedFunction,
+} from "vitest";
+import { SuccessExampleLibraryUseCase } from "../SuccessExampleLibraryUseCase";
+import { SuccessExample } from "../../../domain/entities/SuccessExample";
+import { Genre } from "../../../domain/valueObjects/Genre";
+import { Language } from "../../../domain/valueObjects/Language";
+import { StyleField } from "../../../domain/valueObjects/StyleField";
+import type { ISuccessExampleRepository } from "../../../domain/repositories/ISuccessExampleRepository";
+
+const createMockSuccessExample = (overrides?: Partial<any>) => {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)